Dockerfile.scratch

```diff
@@ -7,11 +7,13 @@ ADD go.* ./
 RUN go mod download
 ADD . .
 RUN make build-static
+RUN install -d -o root -g root -m 1777 /newtmp
```

Comment:
does it have to be root or can we also create a new user?

That would also allow me to configure the helm-chart to run a a certain user and resolve some security issues.

Reply:
Hi @marcofranssen, I responded in the issue you linked to. We want to do this, but we are planning to handle it in a separate PR.